### DISCUSSION TITLE
Output directory for the http-server-csharp emitter

### DISCUSSION THREAD

**Discussion by vbossica** [microsoft/typespec#4093] [General]
Shouldn't `http-server-csharp` also emits its files under `src/Generated`, like `http-client-csharp` does?

**Comment by markcowl** [comment]
Yes, as much as possible, settings should be consistent.  They should both have `emitter-output-dir` settings, although the scenarios for usage are not quite the same.  The client emitter emits a project by default and the service emitter does not, because it is designed around re-generating service code in place.  When the service emitter is enhanced to optionally emit a startup project, the emitted directories will look more similar.
